.gitignore

```diff
@@ -13,6 +13,8 @@
 /bin
 
 # misc
+data/
```

Comment:
does data/ need to be in the gitignore if it's only used within the docker image?
having `data/` as a default data directory would be reasonable (likely much more convenient than status quo) but i'd keep that change to a separate PR. also, once we launch the [sourcecred instance](https://github.com/sourcecred/sourcecred/issues/1232#issuecomment-519538494) concept, we should have a more principled way to do this.

also, can we take `.token` out of the gitignore? i think we decided to keep doing it in the env by default?

(i personally have `SOURCECRED_GITHUB_TOKEN` set by my `.zshrc`, would rather encourage people to take that approach. although i recognize that futzing with zshrc is more of a pain than just writing a file, so I see the appeal to the `.token` approach too.)

Reply:
Data is the suggested folder for the user to bind to the host, so even in the case that only 0.25 of users try it out, it's good to have there. The (internal container) folder can be very specific, but the user is most likely to want to bind somewhere obvious in the present working directory. The instructions say $PWD/data so it's logical to add this, it doesn't hurt but has potential to help.